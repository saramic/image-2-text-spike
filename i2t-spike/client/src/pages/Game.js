import React from 'react'
import { Link } from 'react-router-dom'

const Game = () => (
  <div>
    Game
    <Link to="/rjs-game/round">Round</Link>
  </div>
)

export default Game
