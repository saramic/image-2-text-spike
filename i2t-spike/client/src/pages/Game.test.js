import React from 'react';
import ReactDOM from 'react-dom';
import Game from './Game';
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";

it('<Game /> snapshot', () => {
  const component = renderer.create(<BrowserRouter><Game /></BrowserRouter>);
  expect(component).toMatchSnapshot();
});