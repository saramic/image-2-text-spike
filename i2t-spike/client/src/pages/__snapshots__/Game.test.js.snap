// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> snapshot 1`] = `
<div>
  Game
  <a
    href="/rjs-game/round"
    onClick={[Function]}
  >
    Round
  </a>
</div>
`;
